app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,5 @@
 <resources>
     <string name="app_name" translatable="false">Gnirehtet</string>
     <string name="relay_connected">Reverse tethering enabled</string>
-    <string name="relay_disconnected">Disconnected from the relay server</string>
```

Comment:
I would like the close-on-disconnect behavior *in addition to* the current behavior (via a command-line option). See https://github.com/Genymobile/gnirehtet/issues/51

Reply:
Oh, dude dont find this thread. Think, when device disconnected from relay server, out service and notif only angry users (my users). Notif can not close.

Sorry my english, bro, i from russian,

Reply:
A `-k` option to keep the current behavior would be perfect :)

Reply:
I can work on this option when I have some time, if you prefer.